Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -35,6 +35,7 @@ public struct ConvertAction: Action, RecreatingContext {
     let inheritDocs: Bool
     let treatWarningsAsErrors: Bool
     let experimentalEnableCustomTemplates: Bool
+    let experimentalParseDoxygenCommands: Bool
```

Comment:
I don't think the changes in this file are needed since we're using the global `Features.current` for all the checks. `experimentalEnableCustomTemplates` was added prior to the `Features` struct so it follows a different model.

Reply:
The primary reason i added it here was for the ConvertSubcommandTests test, since i was basically copying the other test that set up the device-frame support and wanted to make sure that the actions could be used in the test without generating the "unused variable" warning.

Reply:
Sorry, I'm not totally following. The DeviceFrame support example doesn't add a property here, does it?

Reply:
It looks like in the time between when i started working on this and when i posted it, the test warning in question has been fixed! 😅 I can take this out.